modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -86,8 +86,7 @@
 
 (defmethod driver/database-supports? [:redshift :convert-timezone]
   [_driver _feat _db]
-  ;; TODO redshift could supports convert-timezone
-  false)
+  true)
```

Comment:
redshift inherits from postgres so this defmethod isn't necessary

Reply:
removed ee2179f55d6448b1153ef4b4b5ce7f73b0a6ec19